src/main/java/frc/robot/RobotContainer.java

```diff
@@ -363,14 +404,20 @@ public void enabledInit() {
         scoringSubsystem.setAction(ScoringAction.WAIT);
     }
 
+    public void onDSConnect() {
+        drive.configurePathPlanner();
+    }
+
     public Command getAutonomousCommand() {
-        return drive.getAutoPath("Example");
+        return drive.getAutoPath("4 Note");
```

Comment:
Can we make this configurable (or make a ticket to do so at competition)?

Reply:
yeah ill add that real quick